integ-test/src/test/java/com/amazon/opendistroforelasticsearch/sql/sql/DateTimeFunctionIT.java

```diff
@@ -60,6 +60,15 @@ public void testDateInGroupBy() throws IOException{
             rows("2018-08-11"));
   }
 
+  @Test
+  public void testDateWithHavingClauseOnly() throws IOException {
```

Comment:
Could you add some more test queries as comparison test for this issue in https://github.com/opendistro-for-elasticsearch/sql/tree/develop/integ-test/src/test/resources/correctness/bugfixes

Reply:
Sure, I will add queries without the date function.

Reply:
I added a few tests in comparison but while testing function without input PI() the results returned but another database was different (3.15 instead of 3.141592653589793) so I added manual IT for this.